deployment/src/test/java/io/quarkiverse/wiremock/devservice/WireMockBasicTest.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkiverse.wiremock.devservice;
+
+import static io.quarkiverse.wiremock.devservice.WireMockConfig.APP_PROPERTIES;
+import static io.quarkiverse.wiremock.devservice.WireMockConfig.PORT;
+import static org.hamcrest.Matchers.is;
+
+import org.eclipse.jetty.server.Response;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+class WireMockBasicTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest UNIT_TEST = new QuarkusUnitTest().withConfigurationResource(APP_PROPERTIES)
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class));
+
+    @Test
+    void testWireMockMappingsFolder() {
+        final String port = ConfigProvider.getConfig().getOptionalValue(PORT, String.class).orElseThrow();
```

Comment:
why is this using `getOptionalValue()`?

Reply:
@wjglerum Thanks for your feedback. I'll look into your comments and incorporated them when applicable.

Reply:
> why is this using `getOptionalValue()`?

Good catch. Actually, there is no need for this.